internal/metrics/client.go

```diff
@@ -69,6 +77,33 @@ func NewClient(url string, timeout time.Duration) (Client, error) {
 	}, nil
 }
 
+func (c *client) Measure(b Benchmarker, f queryFunc, name string,job string, confDescription string, defaultRange model.Duration) error {
+	measure, err := f(job, defaultRange)
+	if err != nil {
+		return fmt.Errorf("queryMetric %s failed job: %s err: %w", name, job, err)
+	}
+	b.RecordValue(fmt.Sprintf("%s - %s - %s",job, name, confDescription), measure)
+	return nil
+}
+
+func (c *client) ProcessCPU(job string, duration model.Duration) (float64, error) {
+	query := fmt.Sprintf(
+		`sum(rate(process_cpu_seconds_total{job="%s"}[%s]) * 1000)`, // in m (i.e. 1000 = 1 vCore)
+		job, duration,
+	)
+
+	return c.executeScalarQuery(query)
+}
```

Comment:
As mentioned below, if we go with optionally requiring cAdvisor we should use `container_cpu_user_seconds_total` instread of `process_cpu_seconds_total`

Reply:
I do not see that metric exposed by the process (same as bellow, I compared with `oc adm top pod` and see same numbers so this metric is good for what we are trying to measure)

Reply:
As per adding cavidsor, we can use `container_cpu_user_seconds_total` now, WDYT?